doc/faq.rst

```diff
@@ -61,6 +61,92 @@ reporting a bug, which should look something like this::
+
+What are all these options for resampling, decimating, and binning data?
+------------------------------------------------------------------------
+
+There are many functions in MNE-Python for changing the effective sampling
+rate of data. We'll discuss some major ones here, with some of their
+implications:
+
+- :func:`mne.io.Raw.resample` is used to resample (typically downsample) raw
+  data. Resampling is the two-step process of applying a low-pass FIR filter
+  and subselecting samples from the data.
+
+  Using this function to resample data before forming :class:`Epochs`
+  for final analysis is generally discouraged because doing so effectively
+  loses precision (and jitters) event timings, see
+  `this gist <https://gist.github.com/Eric89GXL/01642cb3789992fbca59>`_ as
+  a demonstration. However, resampling raw data can be useful for
+  (at least):
+
+    - Computing projectors in low- or band-passed data
+    - Exploring data
+
+- :func:`mne.preprocessing.ICA.fit` has a similar effect as
+  :func:`mne.Epochs.decimate`...
+
+- :func:`mne.Epochs.decimate` function (which does the same thing as the
+  ``decim`` parameter in the :class:`Epochs` constructor) sub-selects every
+  :math:`N^{th}` sample before and after each event. This should only be
+  used when the raw data have been sufficiently low-passed e.g. by
+  :func:`mne.io.Raw.filter` to avoid aliasing artifacts.
+
+- :func:`mne.Epochs.resample`, :func:`mne.Evoked.resample`, and
+  :func:`mne.SourceEstimate.resample` all resample data.
+  This process avoids potential aliasing artifacts because the
+  resampling process applies a low-pass filter. However, this filtering
+  introduces edge artifacts. Edge artifacts also exist when using
+  :func:`mne.io.Raw.resample`, but there the edge artifacts are constrained
+  to two times: the start and end of the recording. With these three methods,
+  edge artifacts are introduced to the start and end of every epoch
+  of data (or the start and end of the :class:`mne.Evoked` or
+  :class:`mne.SourceEstimate` data), which often has a more pronounced
+  effect on the data.
+
+- :func:`mne.SourceEstimate.bin` can be used to decimate, with or without
+  "binning" (averaging across data points). This is equivalent to applying
+  a moving-average (boxcar) filter to the data and decimating. A boxcar in
+  time is a `sinc <https://en.wikipedia.org/wiki/Sinc_function>`_ in
+  frequency, so this acts as a simplistic, non-ideal low-pass filter;
+  this will reduce but not eliminate aliasing if data were not sufficiently
+  low-passed. In the case where the "filter" or bin-width is a single sample
+  (i.e., an impulse) this operation simplifies to decimation without filtering.
+
+Resampling raw data is taking forever! What do I do?
+----------------------------------------------------
+
+:func:`mne.io.Raw.resample` was significantly sped up for version 0.12 by
+using the arameter ``npad=='auto'``. Try it, it might help!
+
+
 Inverse Solution
```

Comment:
Shouldn't this following section (inverse modeling, covariance etc) below be in the tutorial or examples? My expectation is that the FAQ should relate to setup, i/o, memory, classic bugs as opposed to scientific questions.